cookbooks/bcpc-hadoop/recipes/graphite_disk_wrapper.rb

```diff
@@ -0,0 +1,18 @@
+# assign prepare a direcotry structure for graphite to install itself to
+reservation_requests = node[:bcpc][:hadoop][:disks][:reservation_requests]
+
+if reservation_requests.include?("graphite_disk") then
+  disk_index = reservation_requests.index("graphite_disk")
+
+  directory "/disk/#{disk_index}/graphite_disk" do
```

Comment:
More descriptive since we are supposedly reserving a whole disk disk, though in the future we may perhaps have allowances to colocate technologies on same disk by having a list of lists.